roles/kubernetes-apps/ingress_controller/ambassador/tasks/main.yml

```diff
@@ -0,0 +1,57 @@
+---
+
+- name: Ambassador | Create addon dir
+  file:
+    path: "{{ kube_config_dir }}/addons/ambassador"
+    state: directory
+    owner: root
+    group: root
+    mode: 0755
+  when:
+    - inventory_hostname == groups['kube-master'][0]
```

Comment:
Maybe those tests can be in the meta file... As It's not the case for other apps, I'm not sure on this one...

Reply:
@MrFreezeex as you wish, but it seems the other Ingress Controllers are not doing that, right? :thinking:

Reply:
Yes, that's why I am not sure for this change... I would suggest to wait for someone else opinion...

Reply:
Thanks @MrFreezeex. I'm trying to address your other comments meanwhile...